dist/obs-server.spec

```diff
@@ -567,6 +567,15 @@ getent passwd obsapidelayed >/dev/null || \
   /usr/sbin/useradd -r -s /bin/bash -c "User for build service api delayed jobs" -d /srv/www/obs/api -g www obsapidelayed
 %service_add_pre %{obs_api_support_scripts}
 
+# On upgrade keep the values for the %post script
+if [ "$1" == 2 ]; then
+  systemctl --quiet is-enabled obsapidelayed.service && touch %{_rundir}/enable_obs-api-support.target
```

Comment:
wouldnt you need to touch 2 files?

1. is_enabled 
2. is_active

Reply:
done in the next line if "is-active"